server/models/Collective.js

```diff
@@ -2135,17 +2135,31 @@ export default function (Sequelize, DataTypes) {
     });
   };
 
-  Collective.prototype.getBalance = function (until) {
+  Collective.prototype.getBalance = async function (until) {
     until = until || new Date();
-    return models.Transaction.findOne({
+    const transactions = await models.Transaction.findOne({
       attributes: [
         [Sequelize.fn('COALESCE', Sequelize.fn('SUM', Sequelize.col('netAmountInCollectiveCurrency')), 0), 'total'],
       ],
       where: {
         CollectiveId: this.id,
         createdAt: { [Op.lt]: until },
       },
-    }).then(result => Promise.resolve(parseInt(result.toJSON().total, 10)));
+    });
+    const transactionSum = parseInt(transactions.toJSON().total, 10);
```

Comment:
Another way to achieve that is to pass `raw: true` to the query, that way sequelize will return a simple object (rather than a model) and you'll be able to to `transactions.total` directly.